src/mod_last.erl

```diff
@@ -244,6 +251,30 @@ remove_domain(Acc, HostType, Domain) ->
     mod_last_backend:remove_domain(HostType, Domain),
     Acc.
 
+-spec user_receive_iq(mongoose_acc:t(), mongoose_c2s:hook_params(), gen_hook:extra()) ->
```

Comment:
Could you describe in the PR why this new function is needed? I had to look into `ejabberd.c2s` to find the old code that it replaces. Maybe add a PR description or a meaningful commit message?